.github/workflows/build_sqlite_binaries.yml

```diff
@@ -0,0 +1,107 @@
+name: Build SQLite Binaries
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
```

Comment:
As soon as we will be running everything on one machine we can get rid of matrix and just have all config inline rather then through matrix